### ISSUE TITLE
Changing sheet type breaks certain characters (ranged.st is undefined)

### ISSUE THREAD

**Issue by enlorca** [StefanLeng/gurps-categorized-sheet#5]
Hello, when I change character sheets of certain characters I get an error and it doesn't allow me to open the character unless I change the 'sheetClass' value in their json back to "sheetClass": "gurps.GurpsActorSheet".

The error in the console mentions something about 'ranged.st is undefined', but I'm not exactly sure of what it could be. The characters are displaying properly in the default sheets: Maybe the sheet needs to add an exception for that value?

I'll attach the error down here as well as two of the characters that have that issue:
[fvtt-Actor-baraq-S1VKw0XrXxmqA0XC.json](https://github.com/user-attachments/files/17578922/fvtt-Actor-baraq-S1VKw0XrXxmqA0XC.json)
[fvtt-Actor-kade-bonecrusher-BoBV7iynpUmUz0TG.json](https://github.com/user-attachments/files/17578923/fvtt-Actor-kade-bonecrusher-BoBV7iynpUmUz0TG.json)
```


Foundry VTT | Rendering SLCatSheet foundry.js:5826:15
Error: An error occurred while rendering SLCatSheet 418. ranged.st is undefined
[Detected 2 packages: gurps-categorized-sheet(0.3.1), system:gurps(0.17.14)]
    onError foundry.js:654
    onError#0 libWrapper-wrapper.js:188
    render foundry.js:5795
Caused by: TypeError: ranged.st is undefined
[Detected 2 packages: gurps-categorized-sheet(0.3.1), system:gurps(0.17.14)]
    rangedToGrip weaponGrips.ts:179
    rangedGrips weaponGrips.ts:227
    rangedGrips weaponGrips.ts:227
    resolveWeapons weaponGrips.ts:393
    getData cat-sheet.ts:107
    _render foundry.js:5838
    _render foundry.js:6572
    _render foundry.js:7158
    _render actor-sheet.js:54
    render foundry.js:5793
    _onSheetChange foundry.js:11285
foundry.js:655:30
Foundry VTT | Rendering SLCatSheet foundry.js:5826:15
Error: An error occurred while rendering SLCatSheet 421. ranged.st is undefined
[Detected 3 packages: gurps-categorized-sheet(0.3.1), system:gurps(0.17.14), lib-wrapper(1.13.2.0)]
    onError foundry.js:654
    onError#0 libWrapper-wrapper.js:188
    render foundry.js:5795
Caused by: TypeError: ranged.st is undefined
[Detected 3 packages: gurps-categorized-sheet(0.3.1), system:gurps(0.17.14), lib-wrapper(1.13.2.0)]
    rangedToGrip weaponGrips.ts:179
    rangedGrips weaponGrips.ts:227
    rangedGrips weaponGrips.ts:227
    resolveWeapons weaponGrips.ts:393
    getData cat-sheet.ts:107
    _render foundry.js:5838
    _render foundry.js:6572
    _render foundry.js:7158
    _render actor-sheet.js:54
    render foundry.js:5793
    _onClickLeft2 foundry.js:61477
    callback foundry.js:30834
    #handleClickLeft2 foundry.js:31057
    #handleLeftDown foundry.js:31016
    #handlePointerDown foundry.js:30981
    notifyListeners EventBoundary.ts:1454
    notifyTarget EventBoundary.ts:662
    propagate EventBoundary.ts:299
    dispatchEvent EventBoundary.ts:210
    mapPointerDown EventBoundary.ts:683
    mapEvent EventBoundary.ts:231
    onPointerDown EventSystem.ts:373
    addEvents EventSystem.ts:514
    setTargetElement EventSystem.ts:479
    init EventSystem.ts:249
    emit Runner.ts:137
    run StartupSystem.ts:60
    ea Renderer.ts:363
    rh autoDetectRenderer.ts:47
    na Application.ts:78
    #createApplication foundry.js:24720
    initialize foundry.js:24614
    initializeCanvas foundry.js:9270
    setupGame foundry.js:9082
    _initializeGameView foundry.js:10432
    _initializeView foundry.js:10408
    initialize foundry.js:8995
    initialize#0 libWrapper-wrapper.js:188
    <anonymous> foundry.js:95974
    async* foundry.js:95954
foundry.js:655:30
Foundry VTT | Synchronized official game time in 104ms foundry.js:3698:13
Uncaught TypeError: Window.getComputedStyle: Argument 1 is not an object.
[Detected 2 packages: system:gurps(0.17.14), lib-wrapper(1.13.2.0)]
    setPosition foundry.js:6347
    setPosition actor-sheet.js:1532
    _onWindowResize foundry.js:10352
    activateListeners foundry.js:10229
    initialize foundry.js:8992
    initialize#0 libWrapper-wrapper.js:188
    <anonymous> foundry.js:95974
    async* foundry.js:95954
foundry.js:6347:27
    setPosition foundry.js:6347
    setPosition actor-sheet.js:1532
    _onWindowResize foundry.js:10352
    (Async: EventListener.handleEvent)
    activateListeners foundry.js:10229
    initialize foundry.js:8992
    InterpretGeneratorResume self-hosted:1413
    AsyncFunctionNext self-hosted:800
    (Async: async)
    initialize#0 libWrapper-wrapper.js:188
    <anonymous> foundry.js:95974
    InterpretGeneratorResume self-hosted:1413
    AsyncFunctionNext self-hosted:800
    (Async: async)
    <anonymous> foundry.js:95954
```

**Comment by StefanLeng** [comment]
Its the grenades not having a ST stat. I did not expecting that. Should be easy to fix.

**Comment by enlorca** [comment]
I was about to leave a comment about that yeah, it seems that the imported .eqp files seem to give buggy values to certain items.

Thank you!

**Comment by StefanLeng** [comment]
Fixed with 0.3.2
